lib-py/casmutils/xtal/coordinate.py

```diff
@@ -61,7 +119,10 @@ def __eq__(self, other):
         bool
 
         """
-        return self._equals(other)
+        if hasattr(self,'_equals') is False:
+            self._equals=Equals(self._pybind_value,1e-5)
```

Comment:
I think this will have very reasonable behavior. That said, I don't think we should have `1e-5` hard coded into the function itself. We at least need the tolerance to be in the entire coordinate module scope, if not have some sort of global module where we define these things. Use a variable defined somewhere sensible instead of the value itself.